packages/flutter/lib/src/widgets/focus_traversal.dart

```diff
@@ -38,11 +38,24 @@ BuildContext? _getAncestor(BuildContext context, {int count = 1}) {
 void _focusAndEnsureVisible(
```

Comment:
If we're going to make a public value where this is the default, perhaps it makes sense to make this a static public function on FocusTraversalPolicy, called `defaultTraversalRequestFocusCallback`? Then people could implement their callback to call the default if all they want to do is log analytics or something.

Reply:
Perhaps, but I'm partial to keeping this function as it is and duplicating it as a static one on FocusTraversalPolicy since the method is also used in `RequestFocusAction`.